packages/styles/utilities.css

```diff
@@ -70,3 +70,9 @@ body .is--hidden {
   display: flex;
   align-items: center;
 }
+
+ul.semantic-only {
```

Comment:
This class doesn't seem to follow the existing patterns. Is this something we care about?

Reply:
naming is hard. I'm open to suggestions 😄

Reply:
I donno, maybe `SemanticList`? Could even be turned into a component:

```
<SemanticList ordered={false} items={[ 'foo', 'bar' }) />
```

Not sure how useful it'd actually be tho 🤷‍♂️

No need to change this if you don't think it matters. I certainly don't think it does, just something I figured I'd ask about so others could give their opinions.